### PR TITLE
Updated PHPStorm UCT related info after the last release

### PR DESCRIPTION
**Pull request by bohdan-harniuk**
## Purpose of this pull request

This pull request (PR) updates information about UCT feature after the last release of the PHPStorm plugin for the Magento 2.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/ext-best-practices/phpstorm/built-in-upgrade-compatibility-tool.html

### CODE REVIEW

File: src/guides/v2.3/ext-best-practices/phpstorm/built-in-upgrade-compatibility-tool.md [hunk 1]
```diff
@@ -6,22 +6,11 @@ functional_areas:
...
-| 1134 | using @deprecated class |
-| 1334 | using @deprecated interface |
-| 1234 | using @deprecated constant |
-| 1534 | using @deprecated property |
-| 1235 | overriding @deprecated constant |
-| 1535 | overriding @deprecated property |
-| 1337 | inherited from @deprecated interface |
-| 1338 | implemented @deprecated interface |
-| 1439 | call @deprecated method |
+There are types of inspections that are covered by this MVP version:
```
Comment (BarnyShergold) [inline src/guides/v2.3/ext-best-practices/phpstorm/built-in-upgrade-compatibility-tool.md:9]:
```suggestion
There are three types of inspections that are covered by this MVP version:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
One small change

File: src/guides/v2.3/ext-best-practices/phpstorm/built-in-upgrade-compatibility-tool.md (hunk 1 above)
Comment (bohdan-harniuk) [reply src/guides/v2.3/ext-best-practices/phpstorm/built-in-upgrade-compatibility-tool.md]: Thank you! I've done it.

**Comment by bohdan-harniuk** [comment]
Hello, @BarnyShergold!

Could you please proceed with the review?

Thanks, 
Bohdan

**Review by BarnyShergold — APPROVED** [review]
Great!

**Comment by bohdan-harniuk** [comment]
Hello, @BarnyShergold!

If it is okay, why it is not merged?

Thanks, Bohdan

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**